addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/internal/core/LxControlDimmer.java

```diff
@@ -28,7 +28,7 @@
     /**
      * A name by which Miniserver refers to dimmer controls
      */
-    static final String TYPE_NAME = "dimmer";
+    private static final String TYPE_NAME = "dimmer";
```

Comment:
This way this class has two parallel uses, which never collide. Also you have an non-static factory method inside the class itself. So to construct the class you first creat a meaningless instance and then use that instance to create the actual instance. 

Maybe a small (inner) class would be more appropriate, so for example a static inner class with name factory with a parent containing the getType and actual classes implementing the create method.

Reply:
@martinvw not sure if this is what you meant, but please take a look what I did in the recent commit

Reply:
Yes that what I meant